### PR TITLE
Add visible guest warning to encourage login

### PR DESCRIPTION
**Pull request by jryans**
If you happen to get logged out, it might not be very clear what has happened visually. This adds a visible warning to the top of the home page to suggest logging in.

Fixes vector-im/riot-web#7629.

Signed-off-by: J. Ryan Stinnett <jryans@gmail.com>

Home Page Screenshot

### CODE REVIEW

**Comment by jryans** [comment]
Okay, I added a register link as well. Is the text perhaps too wordy? We could use buttons for register and login, instead of sentences for each. I think I have only seen the "button with text label" style inside the DevTools dialogs, so maybe this current approach is good to land now, and we can improve look later...?

2018-11-06 at 20 22

Also, it looks like the React end-to-end tests are failing on the first test "alice signs up". When I try these tests locally, they appear to fail with and without my changes, so perhaps it's a known failure...?

**Comment by turt2live** [comment]
It looks like a great solution to me while we're waiting for the redesign to get fully implemented. I believe @ara4n had some thoughts about how this should be designed, so assigning him to take a look.

The test failure is not known, but expected :( The tests rely on synapse develop to be stable enough to test Riot, however I've been working on a feature for the last month that keeps changing how synapse behaves. This ends up resulting in the tests breaking silently when one side changes before the other. I'll go around and submit the PRs to make it all work again.

**Comment by jryans** [comment]
Back in the original issue vector-im/riot-web#7629, @lampholder is unsure about this approach and thinks we might want to revert the login / register buttons to their previous position instead.
